config/prod/kristen-bucket.yaml

```diff
@@ -0,0 +1,49 @@
+# Provision a general S3 bucket or a Synapse External Bucket
+# http://docs.synapse.org/articles/custom_storage_location.html
+template_path: "remote/s3-bucket-v2.yaml"
+stack_name: "kristen-bucket"
+sceptre_user_data:
+  # (Optional) Synapse IDs for owner.txt file
+  SynapseIDs: ["1968150"]  # Kristen Dang
+parameters:
+  # The Sage deparment (Platform, CompOnc, SysBio, Governance, etc..)
+  Department: "CompOnc"
+  # The Sage project (Infrastructure, amp-ad, ntap, dream, etc..)
+  Project: "TESLA"
+  # The resource owner
+  OwnerEmail: "kristen.dang@sagebase.org"
+  # (Optional) true for read-write bucket, false (default) for read-only bucket
+  AllowWriteBucket: "true"
+
+  # Settings have default values but can be overriden. Set the below
+  # parameters *only* if you want to override the defaults.
+
+  # THIS CURRENTLY ONLY WORKS IN THE us-east-1 REGION!!!!!
+  # Data transfers within the same region between AWS resources are free.
+  # true to restrict downloading files from this bucket to only AWS resources (e.g. EC2 , Lambda) within the same region as this bucket.
+  # This will not allow even the owner of the bucket to download objects in this bucket when not using an AWS resource in the same region!
+  SameRegionResourceAccessToBucket: "true"
+
+  # (Optional) Allow accounts, groups, and users to access bucket. (default is no access).
+  GrantAccess:
+    - "arn:aws:iam::325565585839:root"   # Required ARN for a synapse bucket
+    - "arn:aws:sts::563295687221:assumed-role/sandbox-developer/kristen.dang@sagebase.org"
```

Comment:
@kkdang  This allows you to access the bucket using the AWS CLI.  Please let us know if you want some other use here or none at all.

Reply:
Please add Xengie Doan. I'm not sure yet whether we will use the AWS CLI. Waiting for further instructions from Marco and Bruce on this.

Reply:
@zaro0508  how do you want me to contribute my synapse ID and ARN to this? Should I copy the current file and submit my own PR?

Reply:
Yes, please submit your own PR using this this file as a template @xdoan.  Once your PR is up we can close this one.